jina/clients/mixin.py

```diff
@@ -1,9 +1,12 @@
 from functools import partialmethod, wraps
+import asyncio
+from sgqlc.endpoint.http import HTTPEndpoint as SgqlcHTTPEndpoint
```

Comment:
this `import` is dangerous. This should be hidden with the `ImportExtensions` in a lazy way